test/specs/requests.spec.js

```diff
@@ -33,6 +33,18 @@ describe('requests', function () {
     });
   });
 
+  it('should throw an error if given headers are not an object', function(done) {
```

Comment:
Adding the test here is fine. But in `test/specs/helpers/normalizeHeaderName.spec.js` would be better.